pkg/virt-handler/vm.go

```diff
@@ -2414,6 +2415,49 @@ func (d *VirtualMachineController) vmUpdateHelperDefault(origVMI *v1.VirtualMach
 		}
 	}
 
+	// Find iotune configurations
+	iotuneConf := &cmdv1.DiskIoTune{}
+	for _, v := range vmi.Spec.Volumes {
+		var name string
+		source := v.VolumeSource
+		if source.PersistentVolumeClaim != nil || source.DataVolume != nil {
+			if source.PersistentVolumeClaim != nil {
+				name = source.PersistentVolumeClaim.ClaimName
+			} else {
+				name = source.DataVolume.Name
+			}
+
+			pvc, err := d.clientset.CoreV1().PersistentVolumeClaims(vmi.Namespace).Get(context.Background(), name, metav1.GetOptions{})
+			if err != nil {
+				continue
+			}
+
+			annotations := pvc.ObjectMeta.Annotations
```

Comment:
Looks like CDI datavolume controller will have to be updated to pass iotune annotations to the pvc

Reply:
When I was testing this on my dev machine, I didn't change anything in CDI controller. I applied 2 example YAMLs above and they worked.

FYI, I was looking at [this line](https://github.com/kubevirt/kubevirt/blob/81e8bb646fb3bc3de4b47018acfa5d5e78cdc495/pkg/virt-handler/vm.go#L2411) before I started this PR. It gave me some clue that no update CDI controller is required.

If I misunderstood you, please explain again. Thanks.